pepr/robustness/foolbox_wrapper.py

```diff
@@ -174,15 +189,27 @@ def run(self):
 
             # Calculate average distance of adversarial examples
             dist_eps = []
-            for i in range(len(self.epsilons)):
-                dist_eps.append(fb.distances.l2(inputs_t, raw[i]).numpy().mean())
+            avg_dist_eps = []
+            for e in range(len(self.epsilons)):
+                d = fb.distances.l2(inputs_t, clipped[e]).numpy()
+                avg_dist_eps.append(d.mean())
+                dist_eps_class = []
+                for c in range(np.max(self.labels[indices]) + 1):
+                    c_idx = np.where(self.labels[indices] == c)
+                    if c_idx[0].size == 0:  # numpy 1.19.5 specific
```

Comment:
What means "numpy 1.19.5-specific", do we have to fix numpy to this version?

Reply:
After further investigation, I found this note in the Documentation to **`numpy.where()`**:
> When only *condition* is provided, this function is a shorthand for `np.asarray(condition).nonzero()`.

In these cases, only _condition_ is provided and `nonzero()` returns a tuple. Therefore, this behavior is not 1.19 specific.

Reply:
Nice, thanks for researching :)